client/my-sites/exporter/guided-transfer-card/index.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import { getSiteSlug } from 'state/sites/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+const Feature = ( { children } ) =>
+	<li className="guided-transfer-card__feature-list-item">
+		<Gridicon className="guided-transfer-card__feature-icon" size={ 18 } icon="checkmark" />
+		<span className="guided-transfer-card__feature-text">
+			{ children }
+		</span>
+	</li>;
+
+class GuidedTransferCard extends Component {
+	constructor() {
+		super();
+		this.purchaseGuidedTransfer = this.purchaseGuidedTransfer.bind( this );
+	}
+
+	purchaseGuidedTransfer() {
+		page( `/settings/export/guided/${this.props.siteSlug}` );
```

Comment:
Maybe we could see if there's a way to remove this side-effecting call to outside of the component itself. Maybe a parameter passed in via Redux or through a controller file?